core/components/molecules/tabs/tabs.js

```diff
@@ -129,20 +131,69 @@ class Tabs extends React.Component {
 Tabs.Element = styled.div`
   ${containerStyles};
 `
+Tabs.TabList = styled.ul`
+  display: flex;
+  border-bottom: 1px solid ${colors.base.grayLight};
+`
+
+Tabs.TabListItem = styled.li`
+  &:not(:last-child) {
+    margin-right: ${spacing.large};
+  }
+`
+
+Tabs.TabLink = styled.button`
+  /* Resets button browser styles */
+  background-color: transparent;
+  border: none;
+  padding: ${spacing.small} 0;
+  color: ${colors.link.default};
+  cursor: pointer;
+  border-bottom: 1px solid transparent;
+  margin-bottom: -1px;
+  line-height: 1.3;
```

Comment:
who dis 🤔

Reply:
I wouldn't use the button component, they don't look the same, and if the button changes, then this component gets destroyed.
 
This is a really good case of repetition been better than the wrong abstraction.

What do you mean by "non-standard"? Cosmos has 2 line heights, 1.6 or 1.3. It'll be clearer when we finish the tokens.

Reply:
> even though they are the same HTML element, they are not the same component.

💯 

> Cosmos has 2 line heights, 1.6 or 1.3

wut, really? I didn't know that

Reply:
well, now line-height is all over the place, we have components with line height `1`, or hacked to be the same as the height of the component. That is very wrong.

But hidden under the dust I could uncover that rule, and confirmed it after talking with @landitus 

I am sure it'll be clearer when we have the variable system in place since there will only be 2 global line heights tokens.